docs/installing.rst

```diff
@@ -6,11 +6,9 @@ Installation
 Do I need to install pip?
 -------------------------
 
-pip is already installed if you're using Python 2 >=2.7.9 or Python 3 >=3.4
-binaries downloaded from `python.org <https://www.python.org>`_, but you'll
-need to :ref:`upgrade pip <Upgrading pip>`.
-
-Additionally, pip will already be installed if you're working in a :ref:`Virtual
+pip is already installed if you are using Python 2 >=2.7.9 or Python 3 >=3.4
+binaries downloaded from `python.org <https://www.python.org>`_ (but you would
+need to :ref:`upgrade pip <Upgrading pip>` or if you are working in a :ref:`Virtual
```

Comment:
If we are contracting this, you should remove the parantheses (which you've forgotten to close) and convert the information to two sentences.